srcdeps-core/src/main/java/org/l2x6/srcdeps/core/impl/builder/AbstractMvnBuilder.java

```diff
@@ -64,7 +66,16 @@ public AbstractMvnBuilder(String executable) {
 
     @Override
     protected List<String> getDefaultBuildArguments() {
-        return mvnDefaultArgs;
+        String settingsPath = System.getProperty(Configuration.SRCDEPS_MVN_SETTINGS_PROP);
```

Comment:
> Would it make sense to default to the specified "-s" from the main build? For instance, if I call the main build with mvn -s project-settings.xml clean install, would it make sense to use it instead ?

Yes, that is the way I want to go. Not sure, if this already the case or if there is still something to do so that it works the way you describe (I forgot all the details during my PTO :) . In any case, this should be covered by tests that I plan to write next.